cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/query/IndexCreator.java

```diff
@@ -158,7 +162,7 @@ private String ensureIndexed(Index index) {
             return null;
         }
 
-        final Index indexToCreate = index;
+        final Index proposedIndex = index;
```

Comment:
Ah you got this the wrong way around -- sorry for not being clear. I meant _this_ variable would be called `index` and the argument to `ensureIndexed` would be called `proposedIndex`.